openebs-nativek8s/pipelines/stages/3-functional/ZFS-LocalPV-provisioner/ZFS-LocalPV-provisioner

```diff
@@ -25,6 +25,8 @@ current_time=$(eval $time)
 ## Create the e2e-result-custom-resources for the jobs
 bash openebs-nativek8s/utils/e2e-cr jobname:s3-j1-zfspv-provisioner jobphase:Waiting
 bash openebs-nativek8s/utils/e2e-cr jobname:zfs-ctrl-high-availability jobphase:Waiting
+bash openebs-nativek8s/utils/e2e-cr jobname:zfspv-custom-topology jobphase:Waiting
+bash openebs-nativek8s/utils/e2e-cr jobname:zfspv-raw-block-volume jobphase:Waiting
```

Comment:
is it necessary to pool this job?

Reply:
So the sequence of job will be like this.
zfspv-provisioner ==> zfs-controller in HA ==> After that custom-topology and raw block volume will run in paraller. (As in HA test zfs controller will be down due to some of steps in playbook).
After custom-topology all the jobs are running parallely